SMLHelper/Patchers/CraftDataPatcher_BelowZero.cs

```diff
@@ -39,7 +50,7 @@ private static void AddCustomTechDataToOriginalDictionary()
                 bool techDataExists = TechData.entries.ContainsKey(techType);
                 if (techDataExists && TechData.entries[techType] != CustomTechData[techType])
```

Comment:
`CustomTechData[techType]` is used multiple times in this method.  
A local variable could replace the multiple dictionary lookups with a single one.